packages/ns-ha/README.md

```diff
@@ -132,7 +133,7 @@ echo Nethesis,1234 | ns-ha-config check-backup-node 192.168.100.239
 
 ### Initlialize the primary node
 
-If the requirements are met, you can initialize the primary node:
+If the requirements are met, you can initialize the primary node, please note that the Virtual IP (only) must be written in CIDR notation.
```

Comment:
AFAIK, CIDR notation is not required.

Reply:
It's not strictly required but enforced by the configuration script to avoid issues with routes.